src/common/state/ColorAttribute.xml

```diff
@@ -55,4 +55,7 @@
     <Include file="source" quoted="false">
       cstring 
     </Include>
+    <Include file="header" quoted="false">
```

Comment:
Did you regenerate ColorAttribute with xml2atts?
I ask because you specify `StringHelpers.h` as belonging in the header, but the file that was changed to include this header is ColorAttribute.C

```suggestion
    <Include file="source" quoted="false">
```

Reply:
I tried to do parallel edits to both `.C` file and `.code` file not realizing that this was controlled via `.xml` file until later in the process. I have a build on poodle now and will regen there.